lms/djangoapps/courseware/entrance_exams.py

```diff
@@ -0,0 +1,157 @@
+"""
+This file contains (or should), all entrance exam related utils/logic.
```

Comment:
Optional: This docstring could do without the "(or should)". If you don't feel confident declaring that this file will always contain all entrance exam related utils/logic, then you could reword it to something like "This file contains utils related to entrance exams.".